### PR TITLE
updated excpected outputs for ml03ex03

### PR DESCRIPTION
**Pull request by RobertsOwen**
The tests were done with np.log not both using the epsilon

### CODE REVIEW

File: module08/en.subject.tex [hunk 1]
```diff
@@ -509,7 +509,7 @@ \section*{Examples}
 y_hat1 = logistic_predict(x1, theta1)
 vec_log_loss_(y1, y_hat1)
 # Output:
-0.01814992791780973
+0.018149927917808714
```
Comment (madvid) [inline module08/en.subject.tex:512]: agreed

File: module08/en.subject.tex [hunk 2]
```diff
@@ -518,7 +518,7 @@ \section*{Examples}
 y_hat2 = logistic_predict(x2, theta2)
 vec_log_loss_(y2, y_hat2)
 # Output:
-2.4825011602474483
+2.4825011602472347
```
Comment (madvid) [inline module08/en.subject.tex:521]: I got the following result with my code:
```python
res = 2.482501160247234
```

File: module08/en.subject.tex [hunk 3]
```diff
@@ -518,7 +518,7 @@ \section*{Examples}
...
 
 # Example 3:
```
Comment (madvid) [inline module08/en.subject.tex:523]: Did you check for the example 3 ?
I got the following result with my code:
```python
res = 2.993853310859968
```

**Review by madvid — CHANGES_REQUESTED** [review]
change needed for the 3rd example

File: module08/en.subject.tex (hunk 3 above)
Comment (RobertsOwen) [reply module08/en.subject.tex:523]: you commented twice on the same example with different results on both. which results did you get ?

Comment (madvid) [reply module08/en.subject.tex:523]: No I comment here for the third example. The comment above is for the 2nd example.
I cannot comment the line corresponding to the value of the vec_loss_loss_ output.
So here is for the value on line 530

**Review by madvid — CHANGES_REQUESTED** [review]

File: module08/en.subject.tex (hunk 3 above)
Comment (madvid) [reply module08/en.subject.tex:523]: @RobertsOwen

**Comment by madvid** [comment]
Il y a toujours le 3eme exemple que tu n'as pas corrigé @RobertsOwen

**Review by madvid — APPROVED** [review]
nice !

**Pull request merged**